### PR TITLE
CO : update names & add Cablenoticias

### LINKED ISSUE #7392: Add: Cablenoticias (Bogotà | Colombia)
### Channel Name

Cablenoticias (Bogotà | Colombia)

### Channel ID

Cablenoticias.co

### Website

https://www.cablenoticias.tv

### Stream URL

_No response_

### Notes

Says they stream from their website, but I can't seem to find anything.

YouTube : https://www.youtube.com/cablenoticias

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### PR DESCRIPTION
**Pull request by BellezaEmporium**
Close #7392

### CODE REVIEW

**Comment by LaneSh4d0w** [comment]
Merged the "teveo" PRs. Cuban links should work too.

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked links again.

File: streams/co.m3u [hunk 1]
```diff
@@ -7,22 +7,36 @@ https://movil.ejeserver.com/live/verteve.m3u8
...
+#EXTINF:-1 tvg-id="CanalCaracolAhora.co" status="online",Canal Caracol Ahora (720p)
+https://mdstrm.com/live-stream-playlist/58dc3d471cbe05ff3c8e463e.m3u8
+#EXTINF:-1 tvg-id="CanalDoce.co" status="online",Canal Doce
+https://api.new.livestream.com/accounts/27724665/events/8376416/live.m3u8
 #EXTINF:-1 tvg-id="CanalDos.co" status="online",Canal Dos (Yopal | Casanare) (720p) [Not 24/7]
 http://131.221.42.25:1935/streaming/canal2/playlist.m3u8
 #EXTINF:-1 tvg-id="CanalInstitucionalTV.co" status="online",Canal Institucional TV (1080p)
 https://streaming.rtvc.gov.co/TV_CanalInstitucional_live/smil:live.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="CanalPyC.co" status="online",Canal P&C
+http://66.240.236.26:1935/canalpycvivo/canalpycvivo/canalpycvivo/playlist.m3u8
```
[the remark this answers is not in the document]
Comment (LaneSh4d0w) [reply streams/co.m3u:35]: Used to work yesterday, maybe they do not stream 24/7.

**Review by freearhey — APPROVED** [review]

File: streams/co.m3u (hunk 1 above)
[the remark this answers is not in the document]
Comment (LaneSh4d0w) [reply streams/co.m3u:35]: image

Now works.

**Review by what-a — APPROVED** [review]

**Pull request merged**